src/home_robot_hw/home_robot_hw/nodes/camera_pose_publisher.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+import rospy
+import tf
+from geometry_msgs.msg import PoseStamped
+from home_robot.agent.motion.stretch import STRETCH_CAMERA_FRAME, STRETCH_BASE_FRAME
+from home_robot.utils.pose import to_matrix
+from home_robot_hw.ros.utils import matrix_to_pose_msg
+
+
+class CameraPosePublisher(object):
+    """ publishes the camera pose constantly so that we do not have a dependency on /tf"""
```

Comment:
Why do we want to remove dependency on `tf`?

Reply:
making time synchronization a bit easier